src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainView.scss

```diff
@@ -1,3 +1,5 @@
+@import 'src/styles/common';
```

Comment:
Lots of 3 (or 4?) spaces used as indents in this file. Don't we normally use two spaces?

Reply:
That's done